domain/schema/model/sql/0018-application.sql

```diff
@@ -0,0 +1,107 @@
+CREATE TABLE application (
+    uuid TEXT NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL,
+    life_id INT NOT NULL,
+    charm_modified_version INT,
+    charm_upgrade_on_error BOOLEAN DEFAULT FALSE,
+    exposed BOOLEAN DEFAULT FALSE,
+    placement TEXT,
+    password_hash_algorithm_id TEXT,
+    password_hash TEXT,
+    CONSTRAINT fk_application_life
+    FOREIGN KEY (life_id)
+    REFERENCES life (id)
+);
+
+CREATE TABLE application_channel (
+    application_uuid TEXT NOT NULL PRIMARY KEY,
+    track TEXT NOT NULL,
+    risk TEXT NOT NULL,
+    branch TEXT,
+    CONSTRAINT fk_application_channel_application
+    FOREIGN KEY (application_uuid)
+    REFERENCES application (uuid)
+);
+
+CREATE TABLE application_caas (
```

Comment:
I'd call this `application_scale`, and include a comment that it is only relevant to K8s.

Reply:
Much better, I'll swing back around and fix.